linux_os/guide/services/ftp/ftp_configure_vsftpd/ftp_present_banner/rule.yml

```diff
@@ -1,46 +0,0 @@
-documentation_complete: true
-
-prodtype: fedora,rhel7,rhel8,sle12
```

Comment:
Please don't remove this rule from rhel 7 and rhel 8. Removal of the rule from rhel7 and rhel8 products will cause problems for the downstream users. Instead, you can mark it as deprecated. See  https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rule-deprecation to learn more details about the rule removals and rule deprecation policy. You might have a specific approach regarding rule removal from SLE products, please consult this with a SLE SME.

Reply:
Added a more generic warning macro, to the deprecated rules